### PR TITLE
Add stow_on_each_sample param for Probe

### PR DESCRIPTION
**Pull request by nlef**
I'm used to using this param with BlTouch clone, and i implemented it for generic probe ( BfpTouch micro)

Signed-off-by: Nick Lefskiy n.lefskiy@yandex.ru

### CODE REVIEW

**Comment by KevinOConnor** [comment]
Thanks, but I'm not sure this is a good candidate for the Klipper master branch.  The bltouch devices can use the `[bltouch]` config section.  Other devices can implement what they need using macros.  I'd prefer not to add new user facing options to save a small amount of time on rarely used devices.

-Kevin

**Comment by nlef** [comment]
But BfpTouch ( also known as "poor man bltouch") is quite popular with users of inexpensive (like ender-3) or self-assembled printers. and it'saccuracy is higher. it is more reliable. also this option makes macros easier.

**Comment by nlef** [comment]
maybe it would be more correct to make a separate BfpTouch module by analogy with Bltouch? in this case standart Probe module would have no changes.

**Comment by KevinOConnor** [comment]
Okay, I'll give it some further thought.

Thanks,
-Kevin

File: docs/Config_Reference.md [hunk 1]
```diff
@@ -1584,6 +1588,10 @@ control_pin:
 #pin_move_time: 0.680
 #   The amount of time (in seconds) to wait for the BLTouch pin to
 #   move up or down. The default is 0.680 seconds.
+#   This determines if Klipper should command the pin to move up
+#   between each probe attempt when performing a multiple probe
+#   sequence. Read the directions in docs/BLTouch.md before setting
+#   this to False. The default is True.
```
Comment (KevinOConnor) [inline docs/Config_Reference.md:1594]: This text change looks like a copy-and-paste error.

**Review by KevinOConnor — COMMENTED** [review]
Okay, thanks.  On further thought I think we can add this change.  I did notice one error in the documentation though - see below.

-Kevin

File: docs/Config_Reference.md (hunk 1 above)
Comment (nlef) [reply docs/Config_Reference.md:1594]: I have kept the existing description of the parameter in the documentation. and renamed the parameter in the "probe" section to be logically separate.
I can rewert this back to same naming in both "probe" and "bltouch" sections

Comment (nlef) [reply docs/Config_Reference.md]: Sorry, my mistake. Made a commit with fixes.

**Comment by KevinOConnor** [comment]
Thanks.

-Kevin

**Pull request merged**